### ISSUE TITLE
Feature Request: Show all links in document

### ISSUE THREAD

**Issue by Clock-Drift** [notaapp/nota#110]
Nota currently allows you to view backlinks, however it would also be useful to show all links out to other files in a given document.

**Comment by astoilkov** [comment]
Yes. Totally agree. We've been thinking about this for a while and like the idea a lot. We will do something about this.

Thanks!
